examples/routing_example/public/app.tsx

```diff
@@ -0,0 +1,105 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { AppMountParameters } from 'kibana/public';
+import {
+  EuiPage,
+  EuiPageBody,
+  EuiPageContent,
+  EuiText,
+  EuiHorizontalRule,
+  EuiPageContentHeader,
+  EuiListGroup,
+} from '@elastic/eui';
+import { RandomNumberRouteExample } from './random_number_example';
+import { RandomNumberBetweenRouteExample } from './random_number_between_example';
+import { Services } from './services';
+import { PostMessageRouteExample } from './post_message_example';
+import { GetMessageRouteExample } from './get_message_example';
+
+type Props = Services;
+
+function RoutingExplorer({
+  fetchRandomNumber,
+  fetchRandomNumberBetween,
+  addSuccessToast,
+  postMessage,
+  getMessageById,
+}: Props) {
+  return (
+    <EuiPage>
+      <EuiPageBody>
+        <EuiPageContent>
+          <EuiPageContentHeader>
+            <EuiText>
+              <h1>Routing examples</h1>
+            </EuiText>
+          </EuiPageContentHeader>
+          <EuiText>
+            <EuiListGroup
+              listItems={[
+                {
+                  label: 'IRouter API docs',
```

Comment:
I guess i18n is out of the scope of the example?

Reply:
I'm torn on it.  None of the examples use translations right now. 

I don't think these will ever actually be translated because examples aren't in the distributable (though I could be wrong.  @Bamieh - do you know if there is a way to run via `yarn start` and specify a different language, and have these translations that aren't in the distributable actually translated?).

If you could run it with a different language, it'd be helpful for developers who want to read the examples in a different language (although all of our code comments and READMEs are in English).

It'd be good to have them so copy/pastes will include translations, but this probably wouldn't be code that would be copy/pasted. If a third party developer writes an app and uses these translation services, their code still won't be translated. They need to do extra work.

It's a good question, but until I have clarity, I'm going to leave as is, at least for code that isn't meant to be copy/pasted.